### ISSUE TITLE
Update readme example to be immediately runnable

### ISSUE THREAD

**Issue by thedodd** [thedodd/ybc#9]
Right now there is some content left out for brevity. Seems like that has been tripping-up some folks. Either change the example or make it complete.

**Comment by MarkAnthonyM** [comment]
If pulling from some examples that already exist would work for you and make it easier/save time, maybe you could lift some of the examples from this [reference repo](https://github.com/MarkAnthonyM/ybc-example) I've been putting together? The examples I wrote feel quite long winded though, might not be the most succinct and/or best...

**Comment by quinnjr** [comment]
I've been constantly looking back and forth and the source code for `ybc` since I started incorporating it, and I think the major pain point, at least for me, is the lack of concrete examples in the reference generated over on docs.rs. That's something I could sweep through and try to make more clear over a weekend when I have time.

I'd also suggest doing an `examples` folder of a few simple apps using `ybc`, like a TodoMVC and something showcasing button styling, etc...

**Comment by thedodd** [comment]
Agreed. I think long-term, building a site which somewhat mirrors the Bulma docs using YBC components would be excellent.

That will definitely take a fair bit of time and effort, so starting small with some examples (as you've mentioned) would definitely be a big win.

**Comment by quinnjr** [comment]
I started on a TodoMVC using `ybc` this afternoon. Looks nice, but some changes to `ybc` will be necessary to make it 100% `ybc`-flavored components (eg: adding support for a onkeypress callback on `ybc::Input`)

![Sample](https://i.imgur.com/5KH4DZo.png)
